src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
+  font-family: Roboto, Aria, sans-serif;
+}
+
+:root {
+  --header-active-link-color: #00acdc;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 50px 0;
+}
+
+.nav {
+   margin-top: -10px;
```

Comment:
Avoid using negative margins, as it can lead to overlapping elements and unexpected layout issues. Try to structure the layout to achieve the desired effect without resorting to negative values.